### ISSUE TITLE
Input the same fingerprint image, the similarity score is only 0.97. The same fingerprint image similarity should be 1.

### ISSUE THREAD

**Issue by xuliguo** [XiongjunGuan/JIPNet#2]
Input the same fingerprint image, the similarity score is only 0.97. The same fingerprint image similarity should be 1.

**Comment by XiongjunGuan** [comment]
This is an interesting phenomenon. I did not input the same image as pair when training the network. 
I guess the main goal of network optimization is to make the similarity "sufficiently separable", such as 0.1 and 0.9. That is to say, 0.97 has been considered sufficiently similar in neural networks, and overconfident judgment tendencies may have a negative impact on the testing metrics of other samples.

**Comment by xuliguo** [comment]
> This is an interesting phenomenon. I did not input the same image as pair when training the network. I guess the main goal of network optimization is to make the similarity "sufficiently separable", such as 0.1 and 0.9. That is to say, 0.97 has been considered sufficiently similar in neural networks, and overconfident judgment tendencies may have a negative impact on the testing metrics of other samples.

Thanks，I think it will be forward optimization; this will improve the accuracy of ground truth. You can retrain and verify it;

**Comment by XiongjunGuan** [comment]
Thank you for your suggestion. At present, our work focuses on improving the discriminative performance of genuine / impostor match pairs (as well as relative pose estimation). The interpretability of similarity scores needs further exploration.
